Bottom: 0ff443b76e9380d49aa67614de01a973819800fa
Top:    2edd6d231a54682057fde37240be1342b221fee0
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-03-06 11:51:17 +0800

Refresh of vhost-packed-ring-support

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 1784501..4057a74 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2091,6 +2091,7 @@ static int vhost_get_vq_desc_packed(struct vhost_virtqueue *vq,
 			*out_num += ret;
 		}
 
+		used->wrap_counter = vq->used_wrap_counter;
 		/* On success, increment avail index. */
 		if ((++vq->last_avail_idx & (vq->num - 1)) == 0)
 			vq->used_wrap_counter ^= 1;
@@ -2099,7 +2100,6 @@ static int vhost_get_vq_desc_packed(struct vhost_virtqueue *vq,
 	} while (desc.flags & cpu_to_vhost16(vq, VRING_DESC_F_NEXT));
 
 	used->id = desc.id;
-	used->wrap_counter = vq->used_wrap_counter;
 
 	return 0;
 }
